server/SwipeDecision.py

```diff
@@ -15,16 +16,66 @@ def getPotentialMatchList(currentUserId):
             cursor.execute(PotentialMatchQuery, userID)
             potentialListId = [i[0] for i in cursor.fetchall()]
```

Comment:
instead of just retrieving the ids retrieve their ids AND their longatide and latitude so you don't have to do it again